_data/dataloading/deleted-records.yml

```diff
@@ -0,0 +1,163 @@
+#------------------------------------#
+#      DELETED RECORD EXAMPLES       #
+#------------------------------------#
+
+
+#------------------------------------#
+#        LOG-BASED INCREMENTAL       #
+#------------------------------------#
+
+binlog-replication:
+
+#------------------------------------#
+#       LOG-BASED, HARD DELETES      #
```

Comment:
My understanding was that only the PK value and the _sdc fields would remain when we capture a hard-deletion event with log based incremental. If that's true, it might make sense to add an additional field or two for these examples to show that we clear out those values when we mark the record as deleted, where only the PK and the _sdc fields would remain post delete.

Reply:
Oh really? I didn't know that was the case here. I'll see if we have a test database I can play with to verify this.